tests/integration_tests/scheduler/bin/squeue.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import glob
+import os
+from pathlib import Path
+from typing import Literal, Optional
+
+JobState = Literal["PENDING", "RUNNING", "COMPLETED", "FAILED", "CANCELLED"]
+
+
+def get_parser() -> argparse.ArgumentParser:
```

Comment:
I see that `squeue` and `scontrol` are somewhat new "utilities", it would be great to have them somewhere described briefly. Ie. what they do?

Reply:
Fixed!